carnival.cabal

```diff
@@ -137,9 +139,6 @@ library
                  , yesod-text-markdown
 
 executable         carnival
-    if flag(library-only)
```

Comment:
This felt strange to do, but you can't build the executable in development mode without this change.

I also thought about adding a new executable (`carnival-task` or something), but it seemed wasteful to build two executables with all the same dependencies.